tetris.py

```diff
@@ -177,9 +228,46 @@ def draw_board(canvas):
         x_gap += square_size
 
 
+def get_high_scores():
```

Comment:
This function needs to be renamed. How about `update_high_score_view`?

Reply:
changed to get_high_scores_into_treeview